.github/workflows/create-release.yaml

```
@@ -1,153 +1,301 @@
+         else:
+           print(f"Tag '{tag_name}' does not exists.")
+
+        shell: python
+        continue-on-error: false
+
+#this works only if params.env contains image:tag_version_number
+  update-params-env:
+    runs-on: ubuntu-latest
+    needs: validate-and-fetch-tag
+    outputs:
+      param_env: ${{ steps.read_params_env.outputs.params_env }}
```

actionlint (1.7.8)

property "read_params_env" is not defined in object type {create_tag: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)